### PR TITLE
[Python] Split Python API into files

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
Added more modularity

### CODE REVIEW

File: src/python/py_generation_config.cpp [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        .def_readwrite("frequency_penalty", &GenerationConfig::frequency_penalty)
+        .def_readwrite("rng_seed", &GenerationConfig::rng_seed)
+        .def_readwrite("stop_strings", &GenerationConfig::stop_strings)
+        .def_readwrite("assistant_confidence_threshold", &GenerationConfig::assistant_confidence_threshold)
+        .def_readwrite("num_assistant_tokens", &GenerationConfig::num_assistant_tokens)
+        .def_readwrite("include_stop_str_in_output", &GenerationConfig::include_stop_str_in_output)
+        .def_readwrite("stop_token_ids", &GenerationConfig::stop_token_ids)
+        .def("set_eos_token_id", &GenerationConfig::set_eos_token_id)
+        .def("is_beam_search", &GenerationConfig::is_beam_search);
+}
```
Comment (Wovchena) [inline src/python/py_generation_config.cpp:109]:
```suggestion
}

```

File: src/python/py_openvino_genai.cpp [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    init_llm_pipeline(m);
+
+    // init text2image pipeline
+    init_text2image_pipeline(m);
+
+    // init vlm pipeline
+    init_vlm_pipeline(m);
+
+    // init whisper pipeline
+    init_whisper_pipeline(m);
```
Comment (Wovchena) [inline src/python/py_openvino_genai.cpp:146]: The comments following the the function names exactly look like mocking :)

File: src/python/py_perf_metrics.cpp [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        .def("get_ipot", &PerfMetrics::get_ipot)
+        .def("get_throughput", &PerfMetrics::get_throughput)
+        .def("get_generate_duration", &PerfMetrics::get_generate_duration)
+        .def("get_inference_duration", &PerfMetrics::get_inference_duration)
+        .def("get_tokenization_duration", &PerfMetrics::get_tokenization_duration)
+        .def("get_detokenization_duration", &PerfMetrics::get_detokenization_duration)
+        .def("__add__", &PerfMetrics::operator+)
+        .def("__iadd__", &PerfMetrics::operator+=)
+        .def_readonly("raw_metrics", &PerfMetrics::raw_metrics);
+}
```
Comment (Wovchena) [inline src/python/py_perf_metrics.cpp:158]:
```suggestion
}

```

File: src/python/py_tokenizer.cpp [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            "Override a chat_template read from tokenizer_config.json."
+        )
+
+        .def("get_pad_token_id", &Tokenizer::get_pad_token_id)
+        .def("get_bos_token_id", &Tokenizer::get_bos_token_id)
+        .def("get_eos_token_id", &Tokenizer::get_eos_token_id)
+        .def("get_pad_token", &Tokenizer::get_pad_token)
+        .def("get_bos_token", &Tokenizer::get_bos_token)
+        .def("get_eos_token", &Tokenizer::get_eos_token);
+}
```
Comment (Wovchena) [inline src/python/py_tokenizer.cpp:101]:
```suggestion
}

```

**Review by Wovchena — APPROVED** [review]

File: src/python/py_openvino_genai.cpp (hunk 2 above)
Comment (ilya-lavrenov) [reply src/python/py_openvino_genai.cpp:122]: agree, dropped comments

File: src/python/py_perf_metrics.cpp (hunk 3 above)
Comment (ilya-lavrenov) [reply src/python/py_perf_metrics.cpp]: fixed

File: src/python/py_tokenizer.cpp (hunk 4 above)
Comment (ilya-lavrenov) [reply src/python/py_tokenizer.cpp]: fixed

**Review by popovaan — APPROVED** [review]

**Pull request merged**